### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by derek121**
Fixed a number of typos.

### CODE REVIEW

**Comment by OTP-Maintainer** [comment]
Patch has passed first testings and has been assigned to be reviewed


----------------

I am a script, I am not human 

----------------

File: system/doc/getting_started/robustness.xml [hunk 1]
```diff
@@ -216,7 +216,7 @@ Ping received pong</pre>
       that it does not get killed when it receives abnormal exit
       signals, but all signals will be turned into normal messages on
       the format <c>{'EXIT',FromPID,Reason}</c> and added to the end of
-      the receiving processes message queue. This behaviour is set by:</p>
+      the receiving process's message queue. This behaviour is set by:</p>
```
Comment (nox) [inline system/doc/getting_started/robustness.xml]: `process'`, no?

Comment (zhird) [inline system/doc/getting_started/robustness.xml]: Yes, it should process', not process's.

**Comment by derek121** [comment]
Updated PR with adjusted apostrophe in robustness.xml.

File: system/doc/getting_started/robustness.xml (hunk 1 above)
Comment (ferd) [inline system/doc/getting_started/robustness.xml]: I checked the rules and it depends on the authority. Chicago Manual of Style would say yes, apostrophe only. Others (Oxford University Press, the Modern Language Association, the BBC and The Economist) choose to only use the single possessive apostrophe (`'`) when the word is *plural*. 

In this case, you would write `processes'` but `process's` for the singular form.

I'm personally a fan of Chicago Manual of Style's approach.

**Comment by derek121** [comment]
From @ferd :
```
I checked the rules and it depends on the authority. Chicago Manual of Style would say yes, apostrophe only. Others (Oxford University Press, the Modern Language Association, the BBC and The Economist) choose to only use the single possessive apostrophe (') when the word is plural.

In this case, you would write processes' but process's for the singular form.

I'm personally a fan of Chicago Manual of Style's approach.
```
I'd originally made it "process's", but changed the PR to "process'" based on comments. What's do we say is the verdict?

**Comment by tuncer** [comment]
It's singular, that much is clear. We could use the fact that @joearms is originally from England as a reason to favour Oxford English or English English as some call it :). But Computer English is usually US English, so it's more likely that the rest of the documentation is in US English as well.

Isn't there another typo in the same sentence in
```
 messages on the format {'EXIT',FromPID,Reason}
```
?

**Comment by OTP-Maintainer** [comment]
Patch has passed first testings and has been assigned to be reviewed


----------------

I am a script, I am not human 

----------------

**Comment by zhird** [comment]
We'll use apostrophe only, like the rest of the documentation (for example see the same file, robustness.xml, line 159). Thank you for the contribution!

**Comment by derek121** [comment]
@tuncer - good point- I made that change now.

@zhird - great, then the PR should be good to go as it is now.

Anything I'm missing?

**Comment by OTP-Maintainer** [comment]
Patch has passed first testings and has been assigned to be reviewed


----------------

I am a script, I am not human 

----------------

**Comment by zhird** [comment]
Great, thanks!

**Pull request merged**